src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -6,13 +6,19 @@ export default gql`
     reviewAssignments(condition: { applicationId: $applicationId }, orderBy: TIME_UPDATED_DESC) {
       nodes {
         id
-        levelNumber
         status
         timeUpdated
         levelNumber
         reviewerId
         isLastLevel
         allowedSections
+        stage {
+          id
+          number
+          title
+          colour
+        }
+        stageDate
```

Comment:
was wondering is this the only 'data' addition for this PR ?

Reply:
Yes. Pretty much, I have had to refactor things to also avoid using different types to keep the same data:
* Removed type StageAndStatus
* Replaced StageAndStatus with ApplicationStage in FullStructure of application
* Replace Review.stage from ApplicationStage to new type ReviewStage and move status to inside of that